### ISSUE TITLE
请问如何Finetune CLIP模型？

### ISSUE THREAD

**Issue by awei-97** [CVI-SZU/CLIMS#12]
您好：
      拜读了您的论文，原文中提到：we use the text label descriptions in the training set to finetune the CLIP model (both image and text encoder) for 20 epochs, with an initial learning rate of 0.00005 and a weight decay of 0.003，但是您开源的代码中并没有给出这一过程。您给出了另外一段训练resnet50_cam的代码，但是论文中却并没有提及。我有些困惑，上述哪部分是运行您的算法的必须过程？

      烦请您解答，谢谢

**Comment by Sierkinhane** [comment]
你好！最新版本的CLIMS已经不需要finetune CLIP啦！如果你想微调的话，可以查看`previous_version/`下的代码，里面有包含微调的代码！
BTW, welcome to star our project!
